notify/driver.py

```diff
@@ -85,14 +82,14 @@ class Driver(object):
     }
 
     @classmethod
-    def validate_payload(cls, payload):
```

Comment:
@maretskiy payload is way way way better term than alert https://en.wikipedia.org/wiki/Payload_(computing) and actually i would like to use it so please revert this change. 

Alert is not clear actually in this context